sdk/storage/azure-storage-blob/tests/test_block_blob.py

```diff
@@ -84,11 +84,20 @@ def test_put_block(self, resource_group, location, storage_account, storage_acco
         self._setup(storage_account.name, storage_account_key)
         blob = self._create_blob()
 
+        class _ResponseHandler(object):
```

Comment:
Would be great if you can add a new test instead of modifying existing one,

Reply:
Done